edge/pkg/metamanager/metaserver/server.go

```diff
@@ -266,3 +218,97 @@ func BuildHandlerChain(handler http.Handler, ls *MetaServer) http.Handler {
 	handler = genericfilters.WithPanicRecovery(handler, &apirequest.RequestInfoFactory{})
 	return handler
 }
+
+func (ls *MetaServer) prepareServer() {
```

Comment:
The private function can returns an error to the caller for processing.

Reply:
> The private function can returns an error to the caller for processing.

Done.